elenacaseyroby/YoutubeMusicLib
# New play page view

# What

Updated play page:
- Prints list of videos on search.
- Plays one video from list at a time.
# Why

Trying to move in a direction so this view can be used for searches and playlists.
# Test
- Go to http://localhost:5000/play
- Search for video & play
- Search for another video & play 
- Check db to make sure listens and videos were recorded
- Merge!
